### DISCUSSION TITLE
Compile error.

### DISCUSSION THREAD

**Discussion by Akpodelta3** [OpenFAST/openfast#1301] [Q&A]
Hi,

I am new to extensive programming, but have basic knowledge. OpenFAST fails to run as described in the tutorial. See error message. 

![Compile error for OpenFast V3 2 1](https://user-images.githubusercontent.com/88777077/197912269-f029cbca-fb46-4261-b6f7-1a0eb433c44b.PNG)

**Comment by jjonkman** [comment]
Dear @Akpodelta3,

Does the ElastoDyn input file, "_NRELOffshrBsline5MW_OC4DeepCwindSemi_ElastoDyn.dat_" exist in the same directory as your OpenFAST primary (_.fst_) input file?

Best regards,

**Reply by Akpodelta3** [reply]
Yes it does

**Reply by jjonkman** [reply]
Dear @Akpodelta3,

Can you show how you are calling OpenFAST (what the command line syntax is)?

Best regards,

**Reply by Akpodelta3** [reply]
Here is the details: folder, PowerShell and notepad
![screen_error](https://user-images.githubusercontent.com/88777077/197913724-1d539fd9-260a-4e05-b439-e998c646c2ef.PNG)

**Reply by jjonkman** [reply]
Dear @Akpodelta3,

It looks like your _.fst_ file is indicating that your ElastoDyn input file is one directory back.  Does `EDFile` (line 33) in your _.fst_ file point to the incorrect location of your ElastoDyn file?

Best regards,

**Reply by Akpodelta3** [reply]
Thank you. Here is the .fst file showing the EDFile.
I have viewed other 5MW folders and it seems like the syntax for .fst files are the same.

![EDFile Line 33](https://user-images.githubusercontent.com/88777077/198020755-fadc6838-dd17-44ac-a934-7438e42692e7.PNG)

**Comment by bjonkman** [comment]
@Akpodelta3 , are you able to open and read the ElastoDyn file that OpenFAST is trying to read? I don't see anything obviously wrong with the way the file is specified, so maybe it is a permission issue? Also, sometimes I just copy the file name and paste it into the input file because eyes can play tricks on you--it's sometimes hard to see the difference between some characters.

**Reply by Akpodelta3** [reply]
Thank you for the support. I have adjusted permission settings and done other things.

I will try the installation on a different computer.

Regards.

**Comment by Akpodelta3** [comment]
Hi @jjonkman and @bjonkman,
Thank you for the information. Apparently, during the extraction of the openfast-main, double openfast-main folders resulted in this instance when it worked. This was noticed while working on the installation with my supervisor.
I appreciate. Next is to study the different information in the documentation.

Regards.
![OpenFAST_Working_Result](https://user-images.githubusercontent.com/88777077/199826496-bf2c8af4-59fa-489d-850e-f62f1be3ede3.PNG)
